ArmVirtPkg/ArmVirtQemu.dsc

```diff
@@ -66,7 +66,7 @@
   QemuLoadImageLib|OvmfPkg/Library/GenericQemuLoadImageLib/GenericQemuLoadImageLib.inf
 
   TimerLib|ArmPkg/Library/ArmArchTimerLib/ArmArchTimerLib.inf
-  VirtNorFlashPlatformLib|ArmVirtPkg/Library/NorFlashQemuLib/NorFlashQemuLib.inf
+  VirtNorFlashPlatformLib|OvmfPkg/Library/FdtNorFlashQemuLib/FdtNorFlashQemuLib.inf

```

Comment:
We can remove the old library entirely, no?

Reply:
Yes, the old version can be removed, should I add a patch to remove it?

Reply:
yes please send a follow-up patch to remove it.

Reply:
OK，I will send it today.